### PR TITLE
Add initial version of dd_test

### PR DESCRIPTION
**Pull request by legan4ik**

### CODE REVIEW

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    vm_kp.add_floating_ip(floating_ip)
+    time.sleep(60)
+    ssh_client = ssh.Node('vm', floating_ip.ip,
+                          'ubuntu', pub_key=config['key_path'])
+    ssh_client.run('rm test.dat')
+    ssh_stdout = ssh_client.run('dd if=/dev/zero of=test.dat \
+                                oflag=direct bs={0} count={1}'.
+                                format(count, block_size))
+    global_results.update(request.node.name,
+                          ssh_stdout.split(" ")[-2] +
```
Comment (msenin94) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: please move dd command and logic of parsing to ssh client as separated method `def dd(self, count, block_size):....`

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+@pytest.mark.parametrize(
+    'count,block_size',
+    [('4k', 262144), ('1M', 1024), ('1G', 1)]
+)
+def test_dd(vm_kp, floating_ip, request, global_results,
+            count, block_size):
+    config = utils.get_configuration(__file__)
+    vm_kp.add_floating_ip(floating_ip)
+    time.sleep(60)
```
Comment (msenin94) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: magic constant, let's move it to config file and hence make it configurable

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    ssh_client = ssh.Node('vm', floating_ip.ip,
+                          'ubuntu', pub_key=config['key_path'])
+    ssh_client.run('rm test.dat')
```
Comment (msenin94) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: need to check at least status of command here, just to be ensured

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import mk_verificator.utils.ssh_client as ssh
+import mk_verificator.fixtures.base
+# TODO
+# Create fixture for adding/removing keypair
+# Create fixture for checking security groups
+
+
+@pytest.mark.parametrize(
+    'count,block_size',
+    [('4k', 262144), ('1M', 1024), ('1G', 1)]
```
Comment (msenin94) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: please add more pairs here, you can get all combinations of multiple parametrized arguments, just stack parametrize decorators:
```
@pytest.mark.parametrize("x", [0, 1])
@pytest.mark.parametrize("y", [2, 3])
def test_foo(x, y):
    pass
```

File: mk_verificator/fixtures/base.py [hunk 5]
```diff
@@ -31,10 +33,21 @@ def vm():
...
+    yield vm
+    vm.delete()
+
+
+@pytest.yield_fixture(scope="function")
 def floating_ip():
     mk_nova = nova.Nova()
     floating_ip = mk_nova.client.floating_ips.create(
-                      mk_nova.client.floating_ip_pools.list()[0].name)
+        mk_nova.client.floating_ip_pools.list()[0].name)
```
Comment (msenin94) [inline mk_verificator/fixtures/base.py]: better to move logic of parsing floating list to the up of this function
```
floating_pool = mk_nova.client.floating_ip_pools.list()[0].name
floating_ip = mk_nova.client.floating_ips.create(floating_pool)
```

**Comment by msenin94** [comment]
and, please, fix pep8 checks

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py (hunk 1 above)
Comment (legan4ik) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: https://github.com/msenin94/mk-post-deployment-checks/issues/73

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py (hunk 2 above)
Comment (legan4ik) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: https://github.com/msenin94/mk-post-deployment-checks/issues/74

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py (hunk 3 above)
Comment (legan4ik) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: https://github.com/msenin94/mk-post-deployment-checks/issues/75

File: mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py (hunk 4 above)
Comment (legan4ik) [inline mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py]: Got it. Not need for this particular case

File: mk_verificator/fixtures/base.py (hunk 5 above)
Comment (legan4ik) [inline mk_verificator/fixtures/base.py]: Thanks. Will fix later.

**Pull request merged**